modules/nf-core/jvarkit/vcffilterjdk/main.nf

```diff
@@ -0,0 +1,75 @@
+process JVARKIT_VCFFILTERJDK {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/jvarkit:2024.08.25--hdfd78af_1':
+        'biocontainers/jvarkit:2024.08.25--hdfd78af_1' }"
+
+    input:
+    tuple val(meta),  path(vcf), path(tbi), path(regions_file)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+    tuple val(meta4), path(dict)
+    tuple val(meta5), path(code)
+    output:
+    tuple val(meta), path("*.${extension}"), emit: vcf
```

Comment:
no this option should never be used nowadays because the java lib htsjdk still writes an old deprecated version of the BCF format (and that's why the output of the tool is piped into bcftools view)

Reply:
OK, sounds reasonable